src/main/cpp/t06_homework.cpp

```diff
@@ -80,9 +80,76 @@
 
 #include "t06_homework.h"
 #include <iostream>
+#include <set>
+#include <map>
+#include <string>
+#include <cctype>
 
 using namespace std;
 
 int t06_homework() {
+    set <string> d1;
+    multiset <string> hw1;
+    map <string,string> d;
+    map <string,string> hw;
+    int n,v=0,c,err=0,r=0,b=0,rs=0;;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'rs' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4678801)